// arrays/data_arrays.js
var Colors = ["Red", "Blue", "Yellow",
    "Green", "Brown", "Grey",
    "Black", "White", "Beige",
    "Pink", "Purple", "Orange"];

var Genders = ["Female", "Male"];

var Categories = ["Shoes", "Clothes", "Accessories",
    "Bags", "Underwear", "Tops",
    "Jeans", "Trousers", "Dresses",
    "Sweaters", "Jewelry", "Hats",
    "Gloves", "Coats", "Jackets",
    "Skirts", "Shorts", "Shirts",
    "Blouses", "Belts", "Sportswear"];

var Sizes = ["XXS", "XS", "S",
    "M", "L", "XL",
    "XXL", 35, 36,
    37, 38, 39,
    40, 41, 42,
    43, 44, 45];

var Currencies = ["SEK", "DKK", "NOK",
    "EUR", "GBP", "USD"];

var Countries = ["Sweden", "Norway", "Denmark",
    "United Kingdom", "Finland", "Germany",
    "USA", "France", "Spain"];

var Description = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin rhoncus libero vitae lectus imperdiet, sit amet facilisis nunc pharetra. Nulla eu euismod sem, eu condimentum augue. Donec mattis enim at libero fringilla pulvinar. Fusce in eros sit amet sem tristique sagittis. Nulla euismod, ipsum interdum finibus eleifend, eros ante tempor augue, vitae tempus augue lorem sit amet augue. Suspendisse orci nisl, eleifend non nibh quis, pellentesque finibus dolor. Suspendisse blandit, nulla ac mattis tempor, turpis lectus ultrices sapien, non lobortis arcu velit eu leo. Vivamus rutrum, orci quis dignissim bibendum, est neque molestie sapien, at semper ex nunc eget ligula. Quisque felis libero, auctor ac dictum quis, rhoncus at leo. Integer luctus velit a elit pulvinar fringilla. Ut sit amet accumsan justo, auctor vehicula ex. Mauris vitae mauris aliquam purus sagittis faucibus quis eu urna. Nulla facilisi. Aliquam molestie purus sed convallis laoreet. Vivamus molestie eget felis in fermentum.";

 
exports.Colors = Colors;
exports.Genders = Genders;
exports.Categories = Categories;
exports.Sizes = Sizes;
exports.Currencies = Currencies;
exports.Countries = Countries;
exports.Description = Description;
